enhancements/network/whereabouts-sticky-ips.md

```diff
@@ -0,0 +1,84 @@
+---
+title: whereabouts-sticky-ips
+authors:
+  - "@dougbtv"
+reviewers:
+  - "@s1061123"
+  - "@zshi-redhat"
+approvers:
+  - "@s1061123"
+  - "@zshi-redhat"
+creation-date: 2020-08-05
+last-updated: 2020-08-05
+status: provisional
+---
+
+
+# Whereabouts "Sticky IP addresses"
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This feature implements functionality for Whereabouts where workloads are assigned an IP address, and when those workloads are terminated and restarted, are assigned the same IP address.
+
+## Motivation
+
+This functionality has been requested to provide reliable IP addresses for workloads that advertise their IP address in a method other than Kubernetes services, and where external components require that those same IP addresses be reachable regardless if the workload has been restarted.
+
+This has a benefit for workloads for OpenShift Container Storage (which could utilize this functionality to better provide network isolation by using Whereabouts in tandem with Multus CNI)
+
+### Goals
+
+This should be considered complete if a given workload can be assigned an IP address, be restarted, and still be assigned the same IP address.
+
+### Non-Goals
+
+(No relevant non-goals)
+
+## Proposal
+
+### Implementation Details/Notes/Constraints 
+
+Some upstream discussion regarding the design has already occured in [this upstream issue](https://github.com/dougbtv/whereabouts/issues/51).
+
+Generally the proposed design is to create a new CRD that is specific to an individual IP address. This can be used with, or as an extension to data that exists for storing IP addresses individually in order to determine if they are used in overlapping ranges.
```

Comment:
CRD or CR?

It would be interesting to see how closely that matches https://github.com/metal3-io/metal3-docs/blob/master/design/ip-address-manager/ip-address-management-for-networkdata.md